### ISSUE TITLE
Could you add Transfer Pak support to Mupen64Plus?

### ISSUE THREAD

**Issue by renanr** [OpenEmu/Mupen64Plus-Core#35]
Asking because Mupen64Plus version 2.5.9 supports it, and configuration is done via the config file `mupen64plus.cfg`.

**Comment by behagoras** [comment]
I do have the same request

**Comment by stephenespinal22** [comment]
yes please, I also have the same request! I want to battle with my pokemon from my savs!
